### ISSUE TITLE
support mutlimodul projekte

### ISSUE THREAD

**Issue by hupling** [it-at-m/lhm_actions#42]
![grafik](https://github.com/user-attachments/assets/08ed576d-ec99-482d-8349-bef80ee953ee)

übergebe den ganzen pfad. z. b. `./backend` ` das war früher auch so

**Comment by hupling** [comment]
von @simonhir 
Eine Frage zu den ganzen Action, da hattest Du ja schon mal angesprochen, dass die wegen dem app-path teilweise nur für Mono-Repos gehen.
Habt ihr euch da irgendwie drauf geeinigt, dass man erst mal nur Mono-Repos unterstützt?

**Comment by hupling** [comment]
also im Moment haben wir gesagt, wir fokusieren uns erstmal auf andere Issues für das Release. Wir werden das in einer späteren Version hinzufügen. Es wäre ein Breaking Change

**Comment by simonhir** [comment]
As `//` is parsed correctly, in my opinion there is no need to change anything.
See https://github.com/it-at-m/sps/actions/runs/14513918836/job/40718620826#step:2:128 and https://github.com/it-at-m/lhm_actions/pull/72

**Comment by hupling** [comment]
> > As `//` is parsed correctly, in my opinion there is no need to change anything. See https://github.com/it-at-m/sps/actions/runs/14513918836/job/40718620826#step:2:128 and [#72](https://github.com/it-at-m/lhm_actions/pull/72)
> 
> Seems that action-maven-build works fine also with multi repos, see https://github.com/it-at-m/agile-test-automation-framework/blob/0a8c150332e20f8a25a97b0cdaef8465814acd18/.github/workflows/maven-build.yml#L18

@ejcsid  aber der workflow sieht doch komplett anders aus;

https://github.com/it-at-m/agile-test-automation-framework/blob/0a8c150332e20f8a25a97b0cdaef8465814acd18/.github/workflows/maven-build.yml und https://github.com/it-at-m/refarch-templates/blob/main/.github/workflows/build.yml

**Comment by ejcsid** [comment]
Ja, der workflow, ist anders, aber die Actions funktionieren. Ich weiß nicht, ob wir den Workflow in refarch-templates so implementieren sollen, dass er auch für Multi-Repos funktioniert. Wir können ja ein Beispiel machen, an dem sich diese Projekte orientieren können.
